projects/ad9081/Makefile

```diff
@@ -0,0 +1,15 @@
+TARGET := ad9081
+IIO_SUPPORT = y
+ifeq ($(OS), Windows_NT)
+include ../../tools/scripts/windows.mk
+else
+include ../../tools/scripts/linux.mk
+endif
+ifeq (y,$(strip $(IIO_SUPPORT)))
+	CFLAGS += -D IIO_SUPPORT						\
+		  -D _USE_STD_INT_TYPES						\
+		  -D TINYIIOD_VERSION_MAJOR=0					\
+		  -D TINYIIOD_VERSION_MINOR=1					\
+		  -D TINYIIOD_VERSION_GIT=0x$(shell git rev-parse --short HEAD)	\
+		  -D IIOD_BUFFER_SIZE=0x1000
```

Comment:
All definitions here are made in 
https://github.com/analogdevicesinc/libtinyiiod/blob/master/Makefile
and 
https://github.com/analogdevicesinc/no-OS/blob/master/tools/scripts/linux.mk
if you set `TINYIIOD = y`

Reply:
For debug purposes, I've decided to not include tinyiiod as a library. However, since everything seems to work now, I'll declare TINYIIOD instead.